performance/src/main/java/uk/gov/gchq/palisade/example/perf/actions/CreateAction.java

```diff
@@ -94,14 +95,14 @@ private static void copyDir(final Path src, final Path dest) {
     private boolean createWithPolicyDataset(final PerfFileSet fileSet) {
         ExecutorService tasks = Executors.newFixedThreadPool(NUMBER_OF_THREADS, Util.createDaemonThreadFactory());
 
-        Path smallFile = fileSet.smallFile;
-        Path largeFile = fileSet.largeFile;
-        Path manyDir = fileSet.manyDir;
-        Path masterCopy = manyDir.resolve(String.format(PerfUtils.MANY_DUP_DIR_FORMAT, 0));
+        String smallFile = fileSet.smallFile;
```

Comment:
Why the change of types just to have to create them as new Path(var) below?

Reply:
The new types are coming from the PerfFileSet and these are being separated out for ease of use. The `Path.of()` is only being done for the manyDir.
The manyDir could be converted to a `Path` from the `String` value but the other two are used to create in the `new File(var)`